include/deal.II/meshworker/simple.h

```diff
@@ -107,11 +107,12 @@ namespace MeshWorker
       template<class DOFINFO>
       void assemble(const DOFINFO &info1,
                     const DOFINFO &info2);
-    private:
+    protected:
       /**
        * The global residal vectors filled by assemble().
        */
       AnyData residuals;
+
```

Comment:
Not you but there is a typo in line 112, can you fix it please (residal -> residual)